### DISCUSSION TITLE
is it possible to implement this plugin with tile server?

### DISCUSSION THREAD

**Discussion by marilantagliari** [danwild/leaflet-velocity#79] [Q&A]
My question:

>>> integration with flutter app
i have an app developed in flutter, i have the component called Flutter_Map that load tiles leaflet base by 

    urlTemplate: 'https://example.com/{x}/{y}/{z}',

is it possible to consume this lib from a tile server? how can i do this?

**Comment by brentfraser** [comment]
The short answer is No there is currently no capability in leaflet-velocity to do this.  The longer answer is Yes, but it would take a lot of coding to change leaflet-velocity.
